src/PageContent/Settings/Settings.js

```diff
@@ -0,0 +1,148 @@
+                phone: '206-487-2992',
+                position: 'Vice President'
+            },
+            notifications: [
+                {
+                    type: 'recurring_pickup_request',  // Enums.NotificationType
+                    content: '-L5QoXeC_UrL5tRRED3e'  // key of DeliveryRequest
+                }
+            ],
+            smsNotif: true,
+            emailNotif: true
+        };
+        this.setState({
+            account: account,
+            org: {
+                name: account.name,
+                address: account.address,
+                organizationPhone: account.organizationPhone,
+                numVolunteers: account.numVolunteers || null,
+                notes: account.notes || null,
+                emergencypickup: account.emergencypickup || null
+            },
+            coordinator: {
+                details: account.coordinator
+            },
+            secondaryContact: account.secondaryContact || null
+        });
+    }
+
+    render() {
+        return (
+            <div>
+                {this.state.account ?
+
+                    <div> 
+                        <div className="container">
+                            <OrganizationDetails
+                                account={this.state.org}
+                                isAdmin={this.state.account.user_type}
+                                isEditingOrg={this.state.isEditingOrg}
+                                handleEditOrg={this.handleEditOrg.bind(this)}
+                                handleOrgSave={this.handleOrgSave.bind(this)}
+                            />
+                        </div>
+
+                        <div className="scs-spacing" />
+
+                        <div className="container">
+                            <AccountManager
+                                account={this.state.coordinator}
+                                secondaryContact={this.state.secondaryContact}
+                                isAdmin={this.state.account.user_type}
+                                isEditingAcc={this.state.isEditingAcc}
+                                handleEditAccManager={this.handleEditAccManager.bind(this)}
+                                handleAccSave={this.handleAccSave.bind(this)}
+                            />
+                        </div>
+                    </div>
+
+                    :
+
+                    <div> Loading... </div>
+                }
+            </div>
+        );
+    }
+
+    handleEditOrg() {
+        this.setState({
+            isEditingOrg: true
+        });
+    }
+
+    // Backend TODO: Write data to DB
+    handleOrgSave(newDetails) {
+        this.setState({
+            org: newDetails,
+            isEditingOrg: false
+        });
+    }
+
+    handleEditAccManager() {
+        this.setState({
+            isEditingAcc: true
+        });
+    }
+
+    // Backend TODO: Write data to DB
+    handleAccSave(newDetails, secondaryContact) {
```

Comment:
each sub-content component can handle the saving/writing to db inside themselves

Reply:
Fixed.